signal.c

```diff
@@ -520,6 +520,38 @@ rb_f_kill(int argc, const VALUE *argv)
     return INT2FIX(i-1);
 }
 
+/*
+ *  call-seq:
+ *     Process.kill(pid)    -> boolean
+ *
+ * Checks the given process to see whether it is alive.
```

Comment:
It does not see whether it is *alive*, but will return `true` after the process terminated but before `wait(2)`.